docs/source/examples/managed-jobs.rst

```diff
@@ -452,6 +452,286 @@ To submit the pipeline, the same command :code:`sky jobs launch` is used. The pi
+As a result, except for the 5 completed jobs, 1 job is running on the available worker, while the remaining 4 are in the PENDING state, waiting for the previous job to finish.
+
+Clicking on the pool name will show detailed information about the pool, including its resource specification, status of each worker node, and any job currently running on it:
+
+.. image:: ../images/pool-details.png
+  :width: 100%
+  :align: center
+
+In this example, one worker is ready in Nebius, and another is currently provisioning.  
+The ready worker is running the managed job with ID 10.
+The **Worker Details** section displays the current resource summary of the pool, while the **Jobs** section shows a live snapshot of all jobs associated with this pool, including their statuses and job IDs.
+
+.. tip::
+
+  You can use :code:`sky jobs cancel --pool gpu-pool` to cancel all jobs currently running or pending on the pool.
+
+Update a pool
+~~~~~~~~~~~~~
+
+You can update the pool configuration with the following command:
+
+.. code-block:: yaml
+  :emphasize-lines: 3
+
+  # new-pool.yaml
+  pool:
+    workers: 10
+
+  resources:
+    accelerators: {H100:1, H200:1}
+
+  file_mounts:
+    /my-data-2:
+      source: s3://my-dataset-2/
+      mode: MOUNT
+
+  setup: |
+    # Setup commands for all workers
+    echo "Setup complete!"
+
+.. code-block:: console
+
+  $ sky jobs pool apply --pool gpu-pool new-pool.yaml
+
+The :code:`sky jobs pool apply` command can be used to update the configuration of an existing pool with the same name.  
+In this example, it updates the number of workers in the pool to 10.  
+If no such pool exists, it will create a new one; this is equivalent to the behavior demonstrated in the previous example.
+
+Pools will automatically detect changes in the worker configuration. If only the pool configuration (e.g. number of workers) is changed, the pool will be updated in place to reuse the previous workers; otherwise, if the setup, file mounts, workdir, or resources configuration is changed, new worker clusters will be created and the old ones will be terminated gradually.
+
+
+.. note::
+
+  If there is a :code:`workdir` or :code:`file_mounts` field in the worker configuration, workers will always be recreated when the pool is updated. This is to respect any data changes in them.
+
+Terminate a pool
+~~~~~~~~~~~~~~~~
+
+After usage, the pool can be terminated with the following command:
+
+.. code-block:: console
+
+  $ sky jobs pool down gpu-pool
+  Terminating pool(s) 'gpu-pool'. Proceed? [Y/n]: 
+  Pool 'gpu-pool' is scheduled to be terminated.
+
+The pool will be torn down in the background, and any remaining resources will be automatically cleaned up.
+
+.. tip::
+
+  Autoscaling will be supported in the future, allowing the pool to automatically scale down to 0 workers when no jobs are running, and scale up to the desired concurrency level when new jobs are submitted.
```

Comment:
We can expand this into a **Coming soon** checklist. Include also 8-GPU workers running multiple jobs requesting 1 GPU each.